samples/instances.js

```diff
@@ -15,38 +15,209 @@
 
 'use strict';
 
-function listInstances() {
-  // [START bigtable_list_instances]
-  // Imports the Google Cloud client library
-  const Bigtable = require('@google-cloud/bigtable');
+// Imports the Google Cloud client library
+const Bigtable = require('@google-cloud/bigtable');
 
-  // Creates a client
+async function runInstanceOperations(instanceName, clusterName1, clusterName2) {
   const bigtable = new Bigtable();
 
-  // Lists all instances in the current project
-  bigtable
-    .getInstances()
-    .then(results => {
-      const instances = results[0];
-      console.log('Instances:');
-      instances.forEach(instance => console.log(instance.id));
-    })
-    .catch(err => {
-      console.error('ERROR:', err);
+  console.log(`Creating a PRODUCTION Instance`);
+  // [START bigtable_create_prod_instance]
+  // Creates a Production Instance with the name "ssd-instance"
+  // with cluster name "ssd-cluster", 3 nodes and location us-central1-f
+
+  // Set options to create an Instance
+  const instanceOptions = {
+    clusters: [
+      {
+        name: clusterName1,
+        nodes: 3,
+        location: 'us-central1-f',
+        storage: 'ssd',
+      },
+    ],
+    type: 'PRODUCTION', // Optional as default type is PRODUCTION
```

Comment:
@sduskis uppercase or lowercase doesn't matter here, should we prefer upper or lower in the samples?

Reply:
I think that upper case looks better for "constants"